apps/storybook-react-native/.storybook/config.js

```diff
@@ -15,10 +12,19 @@ addons.setConfig({
   },
 });
 
+const ThemeDecorator = ({ children }) => {
+  const colorScheme = useColorScheme();
+  const theme = colorScheme === 'dark' ? Theme.Dark : Theme.Light;
+
+  return (
+    <ThemeProvider theme={theme}>
+      <FontLoader>{children}</FontLoader>
+    </ThemeProvider>
+  );
+};
+
```

Comment:
Refactoring Theme decorator to offload the responsibility of setting the theme at the consumer level